src/rsa.rs

```diff
@@ -100,29 +126,23 @@ macro_rules! rsa_pkcs1 {
         /// Not available in `no_heap` mode.
         pub static $VERIFY_ALGORITHM: signature::VerificationAlgorithm =
                 signature::VerificationAlgorithm {
-            implementation: &RSA_PKCS1 {
-                digest_alg: $digest_alg,
+            implementation: &RSAVerificationAlgorithm {
+                padding_alg: &$PADDING_ALGORITHM,
                 min_bits: $min_bits,
-                digestinfo_prefix: $digestinfo_prefix,
             }
         };
     }
 }
 
-rsa_pkcs1!(RSA_PKCS1_2048_8192_SHA1_VERIFY, 2048, "2048", "SHA-1",
-           &digest::SHA1, &SHA1_PKCS1_DIGESTINFO_PREFIX);
+rsa_pkcs1!(RSA_PKCS1_2048_8192_SHA1_VERIFY, 2048, "2048", "SHA-1", RSA_PKCS1_SHA1);
```

Comment:
These are all things we should figure out in follow-up patches:

Should we rename these to have the pattern `{padding}`_{min}_{max}_VERIFY'? Now it seems weird for the key size to be in the middle.

And/or, if we start all padding names with `PADDING_` and all verify algorithm names with `VERIFY_` then they'll sort separately.

We are not enforcing minimum/maximum key size for signing like we are for verification. IMO, we should have a way of enforcing the same 2048-8192 and 3072-8192 ranges for signing.